src/components/hooks/useFreshInstall.js

```diff
@@ -15,6 +19,18 @@ const installStatus = new MMKV( {
 const INSTALL_STATUS = "alreadyLaunched";
 
 const useFreshInstall = ( currentUser: ?Object ) => {
+  const { i18n } = useTranslation( );
+
+  // only use system locale if this is a fresh install of the app;
+  // otherwise, locale should be set by LanguageSettings with a
+  // fallback to the web locale
+  const changeToSystemLocale = useCallback( ( ) => {
+    const systemLocale = getInatLocaleFromSystemLocale();
+    i18n.changeLanguage( systemLocale );
+  }, [
+    i18n
+  ] );
+
```

Comment:
The other time we want to use the system locale is when the user signs out, and this is not handling that.